pkg/virtiofs/virtiofs.go

```diff
@@ -10,8 +9,3 @@ import (
 // This is empty dir
 var VirtioFSContainers = "virtiofs-containers"
 var VirtioFSContainersMountBaseDir = filepath.Join(util.VirtShareDir, VirtioFSContainers)
-
-func VirtioFSSocketPath(volumeName string) string {
-	socketName := fmt.Sprintf("%s.sock", volumeName)
```

Comment:
I wondering if this file really makes sense. @xpivarc what was the reason to introduce it? Should we merge this and the `pkg/virt-controller/services/virtiofs.go` ? (Not necessarily in this PR)

Reply:
Smaller packages are preferred for many reasons. This package contains virtiofs related logic that needs to be shared between multiple components.

Reply:
Ok, yes make sense but then I'd rather keep the function `VirtioFSSocketPath` and not moving it into virt-controller

Reply:
I didn't move `VirtioFSSocketPath` function, it was duplicated and the function in `pkg/virtiofs/virtiofs.go` was unused.
I agree to merge both files, since I don't see any benefit to have the functionality split in two files. So, why keeping just:

```
var VirtioFSContainers = ...
var VirtioFSContainersMountBaseDir ..
func VirtioFSSocketPath(volumeName string) string ...
```
and the rest of functions in `pkg/virt-controller/services/virtiofs.go`? If we want smaller packages, why not just everything in `pkg/virtiofs/virtiofs.go`? 

Also, just by curiosity, why virtiofsd is part of virt-controller instead of virt-launcher?

Reply:
>I didn't move VirtioFSSocketPath function, it was duplicated and the function in pkg/virtiofs/virtiofs.go was unused.

Correct and Imho this is wrong. It should be other way around the `VirtioFSSocketPath` should be used from the virtiofs package.

>and the rest of functions in pkg/virt-controller/services/virtiofs.go? If we want smaller packages, why not just everything in pkg/virtiofs/virtiofs.go?

As the path suggests "virt-controller", there should be only code/packages relevant to the virt-controller component.
I don't necessarily agree that everything should be in one package if it is going to be used only in one of the components.

>Also, just by curiosity, why virtiofsd is part of virt-controller instead of virt-launcher?

Because virt-controller is responsible for rendering the Pod spec and virt-launcher is part of the Pod spec.

Reply:
Ok, I'll move `VirtioFSSocketPath()` to `pkg/virtiofs/virtiofs.go`